CHANGELOG.md

```
@@ -2,6 +2,9 @@
 
 ## v1.0.1 (unreleased)
 
+#### Improvements
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)